x-pack/test/security_solution_api_integration/README.md

```diff
@@ -111,7 +111,41 @@ In this project, you can run various commands to execute tests and workflows, ea
          ```shell
          npm run initialize-server:dr:default exceptions/workflows ess   
          ```
-      5. **Run tests for "exception_workflows" using the ess runner in the "essEnv" environment:**   
+      5. **Run tests for "exception_workflows" using the ess runner in the "essEnv" environment:**
          ```shell
          npm run run-tests:dr:default exceptions/workflows ess essEnv
-      ```
\ No newline at end of file
+         ```
+
+## Testing with serverless roles
+
+The `supertest` service is logged with the `admin` role by default on serverless. Ideally, every test that runs on serverless should use the most appropriate role.
+
+The `securitySolutionUtils` helper exports the `createSuperTest` function, which accepts the role as a parameter.
+You need to call `createSuperTest` from a lifecycle hook and wait for it to return the `superset` instance.
+All API calls using the returned instance will inject the required auth headers.
+
+**On ESS, `createSuperTest` returns a basic `supertest` instance without headers.*
+
+```js
+export default ({ getService }: FtrProviderContext) => {
+   const utils = getService('securitySolutionUtils');
+
+   describe('@ess @serverless my_test', () => {
+      let supertest: TestAgent;
```

Comment:
Would it me useful to show the type import for `TestAgent`?

Reply:
Yes, that makes sense! I will update the docs.

Reply:
Done!